ct-aa/JavaScript30
# ✨ JS30-3: Add Function to Update CSS Variables

### Related Tickets
- JS30 Day 3 Challenge

### Description
Adds Function to Update CSS Variables to adjust the padding of the image, image blur, background color of the image, and color of the word JS in the Header.

### Validation
1. Open index-START.html
2. Check the index-START.html on the browser.
3. The Padding of the image should change when the "Spacing slider" is dragged.
4. The Image should blur when the "Blur slider" is dragged.
5. The Image "background-color" and the word "JS" in the header should change color when "Base Color" is selected and the color is picked.

### Risks
N/A
